canonical-websites/blog.ubuntu.com
# 346 whats coming up

## Done

Update signup forms for GDPR

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- Go to <http://0.0.0.0:8023/> and <http://0.0.0.0:8023/home> and see that the checkbox text has been updated
